packages/components/src/Disclosure/Disclosure.tsx

```diff
@@ -54,26 +92,77 @@ export function Disclosure({
   const isBelowBreakpoint = exactWidth && exactWidth < Breakpoints.small;
   const isTitleString = typeof title === "string";
 
+  const containerClassNames = classnames(
```

Comment:
Although we have a `details` element and reference `details` in styles, I do feel like it's more intuitive for users for this to be called `container`. `details` is fine internally but I feel like users would confused that with `content`

Reply:
I think container makes sense as well.